Sources/Examples/Echo/Model/echo.grpc.swift

```diff
@@ -149,3 +149,12 @@ extension Echo_EchoProvider {
   }
 }
 
+extension Echo_EchoRequest: Payload {
```

Comment:
This file is generated, but it looks like these were added manually.

The implementation for the conformance to `Payload` _shouldn't_ be in the generated code. Instead we can create another protocol (e.g. `GRPCProtobufPayload`) which conforms to `Payload` and `Message`. It should then provide the default implementation for `Payload` so that the generated messages can be extended to conform to that protocol.

You'll need to take a look at the code-gen (in `protoc-gen-grpc-swift`) to figure out how to do this.

Reply:
@glbrntt Yes, I just added them to see if they would pass the test cases and to have a draft of the PR. so we can talk about What we would be doing, and how.

we can have the following:
```swift
protocol GRPCProtobufPayload: Payload {}
extension GRPCProtobufPayload {
// comes the implementation of the functions
}
```
and similarly in FlatBuffers:
```swift
protocol GRPCFlatbufPayload: Payload {}
extension GRPCFlatbufPayload {
// comes the implementation of the functions
}
```
do you think we should generate the code? or implement it which the package itself?

Reply:
Ah okay, no problem. Just an FYI if you want to run the tests locally you can set the `ENABLE_TIMING_TESTS` environment variable to `false` to skip some of the longer running tests (it takes the test time down to a couple of seconds).

The default implementation for `GRPCProtobufPayload` (which should also conform to `SwiftProtobuf.Message`) should be within `GRPC` since we already depend on `SwiftProtobuf`. This gives us more flexibility to change the implementation (i.e. if/when new Protobuf APIs become available) without regenerating the code.

For flatbuffers I think we _have_ to generate the conformance to `GRPCPayload` each time since there's no dependency between flatbuffers and gRPC Swift here.